cico_run_pydoc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+prep() {
+    yum -y update
+    yum -y install epel-release
+    yum -y install python36 python36-virtualenv which
+}
+
+check_python_version() {
+    python3 tools/check_python_version.py 3 6
```

Comment:
I have always wondered why we use this long script instead of parsing the output of "Python --version" with a regex.

Reply:
@rootAvish , I just replaced it with a simple shell script.